charts/hatchet-api/templates/deployment.yaml

```diff
@@ -71,18 +71,9 @@ spec:
           mountPath: {{ .mountPath }}
           subPath: {{ .subPath }}
 {{- end }}
-        # livenessProbe:
-        #   httpGet:
-        #     path: /api/v1/livez
-        #     port: {{ .Values.service.internalPort }}
-        #   initialDelaySeconds: 15
-        #   periodSeconds: 20
-        # readinessProbe:
-        #   httpGet:
-        #     path: /api/v1/readyz
-        #     port: {{ .Values.service.internalPort }}
-        #   initialDelaySeconds: 5
-        #   periodSeconds: 10
+{{- if .Values.health.enabled }}
```

Comment:
I don't understand why but the healthcheck is somehow always passed, so it is also attached to the `api` which then fails. :/

Reply:
even with https://github.com/hatchet-dev/hatchet-charts/pull/3/commits/7550e70196ac60d05ac240469fd5249f44fcac61 this still happens